docs/guide/examples.rst

```diff
@@ -729,6 +729,16 @@ to keep track of the agent progress.
   model.learn(10_000)
 
 
+SB3 with EnvPool or Isaac Gym
+-----------------------------
+
+Just like Procgen (see above), `EnvPool <https://github.com/sail-sg/envpool>`_ and `Isaac Gym <https://github.com/NVIDIA-Omniverse/IsaacGymEnvs>`_ accelerate the environment by
+already providing a vectorized implementation.
+
+To use SB3 with those tools, you must wrap the env with a ``VecEnvWrapper`` that will pre-process the data for SB3,
```

Comment:
Hmm shouldn't the instruction say that to use SB3 with those tools, you need to wrap it with the tool's version of ``VecEnvWrapper`` (i.e. this `VecEnvWrapper` is not part of SB3)? It currently sounds like it is part of SB3 and you need to use it on top of whatever the tool will provide.

Reply:
true, I should be more specific